doc/articles/Uno-UI-Performance.md

```diff
@@ -17,7 +17,7 @@ Here's what to look for:
 	- The default [ListViewItem and GridViewItem styles](https://github.com/unoplatform/uno/blob/74b7d5d0e953fcdd94223f32f51665af7ce15c60/src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml#L951) are very feature rich, yet that makes them quite slow. For instance, if you know that you're not likely to use selection features for a specific ListView, create a simpler ListViewItem style that some visual states, or the elements that are only used for selection.
 	- If items content frequently change (e.g. live data in TextBlock) on iOS and Android, ListView items rendering can require the use of the `xamarin:AreDimensionsConstrained="True"` [uno-specific property](https://github.com/unoplatform/uno/blob/7355d66f77777b57c660133d5ec011caaa810e29/src/Uno.UI/UI/Xaml/FrameworkElement.cs#L86). This attribute prevents items in a list from requesting their parent to be re-measured when their properties change. It's safe to use the `AreDimensionsConstrained` property when items always have the same size regardless of bound data, and the items and list are stretched in the non-scrolling direction. If item sizes can change when the bound data changes (eg, if they contain bound text that can wrap over multiple lines, images of undetermined size, etc), or if the list is wrapped to the items, then you shouldn't set `AreDimensionsConstrained` because the list does need to remeasure itself when item data changes in that case.
 
-	You'll need to set the property on the top-level element of your item templates, as follows:
+	  You'll need to set the property on the top-level element of your item templates, as follows:
 		```xaml
```

Comment:
Looks like this is the only usage of xaml as codeblock identifier, all other doc pages use xml. I suspect that xaml isn't well-recognized and won't have syntax-highlighting. Can you confirm?

Reply:
confirmed, `xaml` isn't recognised

```suggestion
		```xml
```

Reply:
`xaml` is recognized as a specifier, how did you validate it ?

Reply:
if you build with docfx, it will not show any highlighting